internal/controller/drcluster_controller.go

```diff
@@ -425,6 +431,8 @@ func (r DRClusterReconciler) processCreateOrUpdate(u *drclusterInstance) (ctrl.R
 
 	if reason, err := validateS3Profile(u.ctx, r.APIReader, r.ObjectStoreGetter, u.object, u.namespacedName.String(),
 		u.log); err != nil {
+		u.setDRClusterAvailableStatusMetric(err)
```

Comment:
When is it reset back from `err` to healthy? IOW, there should be a call to something like setDRClusterAvailableStatusMetric(nil), that sets the gauge back to `1` I would assume.

For example, it the call to validateS3Profile succeeds then the gauge should be set to `1`

Reply:
I will make the change